charts/stable/kube-ops-view/values.yaml

```
@@ -0,0 +1,43 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: hjacobs/kube-ops-view
+  # -- image tag
+  tag: 20.4.0
```

```suggestion
  # -- image tag
  # @default -- chart.appVersion
  tag:
```
This makes it so that we only have to update the appVersion (by default) but it still allows users to override it if they want.